app/src/main/AndroidManifest.xml

```diff
@@ -14,22 +14,13 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         android:name=".MyApplication">
-        <activity android:name=".MainActivity"
```

Comment:
You can actually delete the MainActivity file since we're not using it anymore.

Reply:
This is all set!